lib/assert/imgLoaded_client.js

```diff
@@ -0,0 +1,35 @@
+/* jshint browser: true */
+'use strict';
+
+// returns true if the image is loaded and decoded,
+// or a number, if it didn't find one single image,
+// or a helpful error if it wasn't an image,
+// or the path, if it found some non-loaded image
+module.exports = function imgLoaded(selector) {
+  function describe(elem) {
+    var tag = (elem.tagName || '').toLowerCase();
+    if (elem.id) {
+      tag += '#' + elem.id;
+    }
+    var classes = elem.className.replace(/^\s+|\s+$/g, '');
```

Comment:
We are injecting this into the page under test, so polyfilling `.trim` sounds dangerous.